airflow/providers/google/cloud/triggers/bigquery.py

```diff
@@ -148,6 +163,7 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
                 "table_id": self.table_id,
                 "poll_interval": self.poll_interval,
                 "impersonation_chain": self.impersonation_chain,
+                "cancel_on_kill": self.cancel_on_kill,
```

Comment:
Is this needed

Reply:
I think this is needed for line 135

Reply:
Yes we cancel the job only when `cancel_on_kill` is True